tests/integration.rs

```diff
@@ -5467,7 +5467,7 @@ mod tests {
                 guest.ssh_command("screen -dmS reboot sh -c \"sleep 5; echo s | tee /proc/sysrq-trigger; echo c | sudo tee /proc/sysrq-trigger\"").unwrap();
 
                 // Allow some time for the watchdog to trigger (max 30s) and reboot to happen
-                thread::sleep(std::time::Duration::new(50, 0));
+                guest.wait_vm_boot(Some(50)).unwrap();
```

Comment:
@rbradford With the same time-out `50s`, I am not sure whether `wait_vm_boot()` w/ will reduce flaky failures here, as I expect it would only save some unnecessary waiting time (and report errors if vm is not rebooted in 50s). 

I think we need to increase the timeout for reducing the flaky failures as a result of sporadic slow reboot.